### ISSUE TITLE
dyld: Library not loaded:libSwiftSyntax.dylib

### ISSUE THREAD

**Issue by smslit** [Flinesoft/BartyCrouch#116]
After installing **bartycrouch 4.0.0** with **mint**, I called **bartycrouch** in terminal, but got error:

```Bash
➜  ~bartycrouch
dyld: Library not loaded: /private/var/folders/_5/ngh90rqn30s3ym8pwqly3r6h0000gq/T/mint/github.com_Flinesoft_BartyCrouch/.build/x86_64-apple-macosx10.10/release/libSwiftSyntax.dylib
  Referenced from: /usr/local/bin/bartycrouch
  Reason: image not found
[1]    28733 abort      bartycrouch
```

**Comment by Dschee** [comment]
Thank you for reporting this. I actually came across this issue during development, too, but thought I fixed it sometime in the middle. This is a known problem which can be tracked in [this related issue](https://github.com/yonaskolb/Mint/issues/120) on Mint.

Please consider using BartyCrouch via Homebrew until this is fixed. The steps required:

```bash
mint uninstall Flinesoft/BartyCrouch
brew install bartycrouch
```

**Comment by smslit** [comment]
@Dschee I got it, thanks a lot

**Comment by Dschee** [comment]
Please note that libSwiftSyntax is no longer a dependency since version 4.0.1 because Swift 5 already includes it. Therefore this issue should be resolved.

But note that when installing via Mint, you will probably come across [this issue](https://github.com/yonaskolb/Mint/issues/129) now, which I expect to be resolved soon though, now that Xcode 10.2 is out. Until that is fixed, you might still need to install BartyCrouch via Homebrew for the maean time.
